.pre-commit-config.yaml

```diff
@@ -27,3 +27,9 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.982
+    hooks:
+    -   id: mypy
+        pass_filenames: false
```

Comment:
`pass_filenames` is false here, so we run mypy with the `files` every time:

https://github.com/celery/celery/blob/f8aebff8b068fd383ce5c54311f049a06ccad563/pyproject.toml#L15-L20